app/Http/Controllers/API/InviteController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\CheckEmailRequest;
+use App\Services\InviteService;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+
+class InviteController extends Controller
+{
+    public function listOfInvitedUsers(): JsonResponse
+    {
+        $employeeVendor=InviteService::invitedList();
+        return response()->json([
+            'total' => count($employeeVendor),
+            'message'=>'List Of Invited User',
+            'invitedUser'=>$employeeVendor
+
+
+        ],200);
+    }
+    public function reInvite(CheckEmailRequest $request, InviteService $inviteService): JsonResponse
```

Comment:
@Babita3331 In some methods you are using the static methods from services and here you are injecting the service into the method. Please stick to one style and maintain it throughout the codebase